tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -387,6 +387,10 @@ def _augment_configs_with_checkpoint_info(self):
             for tensor_name, tensor_shape in var_map.items():
                 if len(tensor_shape) != 2:
                     continue
+                # Optimizer slot values are the same shape as embeddings
+                # but are not embeddings.
+                if ".OPTIMIZER_SLOT" in tensor_name:
```

Comment:
I realize this is a small tweak, but since it doesn't appear to be related to the main change, and it's a functional change (as opposed to just refactoring or formatting), it would be ideal to break this out as its own PR.

Reply:
👍

Reply:
#3679